### PR TITLE
Fix logrotate

### PR DESCRIPTION
**Pull request by ro-tex**
Install `cron` and make sure it's running. Run `logrotate` hourly instead of daily.

There's a little twist in the Alpine containers in order to avoid extra complexity in `run.sh`.

### CODE REVIEW

**Review by mrcnski — APPROVED** [review]
I'm a little surprised these systems don't come with cron preinstalled, looks good though.

File: dev-debian/Dockerfile [hunk 1]
```diff
@@ -23,7 +23,8 @@ ARG SIA_DIR="/sia"
 ARG SIA_DATA_DIR="/sia-data"
 ARG SIAD_DATA_DIR="/sia-data"
 
-RUN apt-get update && apt-get install -y mime-support logrotate
+RUN apt-get update && apt-get install -y mime-support logrotate cron && \
+    mv /etc/cron.daily/logrotate /etc/cron.hourly
```
Comment (interstella-5555) [inline dev-debian/Dockerfile:27]: I don't think we need to run it hourly, daily should be fine right?

File: dev-debian/Dockerfile [hunk 2]
```diff
@@ -23,7 +23,8 @@ ARG SIA_DIR="/sia"
 ARG SIA_DATA_DIR="/sia-data"
 ARG SIAD_DATA_DIR="/sia-data"
 
-RUN apt-get update && apt-get install -y mime-support logrotate
+RUN apt-get update && apt-get install -y mime-support logrotate cron && \
```
Comment (interstella-5555) [inline dev-debian/Dockerfile:26]: is `cron` not available? I was able to run it manually, it might be just not running but it should be available

Comment (ro-tex) [reply dev-debian/Dockerfile:26]: Depends on the image. I ran into the case where it wasn't installed. It doesn't hurt to make sure.

File: dev-debian/Dockerfile (hunk 1 above)
Comment (ro-tex) [reply dev-debian/Dockerfile:27]: Once a day is mostly OK but with some log spam we might get burnt.

**Pull request merged**